hack/e2e/eksctl.sh

```diff
@@ -63,15 +63,16 @@ function eksctl_create_cluster() {
 
   if [[ "$WINDOWS" == true ]]; then
     ${BIN} create nodegroup \
-      --managed=false \
+      --managed=true \
       --cluster="${CLUSTER_NAME}" \
-      --node-ami-family=WindowsServer2019FullContainer \
+      --node-ami-family=WindowsServer2022FullContainer \
       -n ng-windows \
-      -m 1 \
-      -M 1 \
+      -m 3 \
+      -M 3 \
       --ssh-access \
       --ssh-public-key "${SSH_KEY_PATH}".pub
-    ${BIN} utils install-vpc-controllers --cluster="${CLUSTER_NAME}" --approve
+
+    kubectl apply --kubeconfig "${KUBECONFIG}" -f "$VPC_CONFIGMAP_FILE"
```

Comment:
This file should contain `VPC_CONFIGMAP_FILE=${11}` after line 25

Reply:
✅